src/main/java/hudson/plugins/ircbot/v2/IRCColorizer.java

```diff
@@ -30,7 +30,7 @@ public static String colorize(String message){
             return message;
         } else {
             String line = colorForBuildResult(message);
-            if (line == message) { // line didn't contain a build result
+            if (line.equals(message)) { // line didn't contain a build result
```

Comment:
What if `line` is `null`? I was elsewhere advised to use comparisons rather than `.equals()` with a risk of NPE (or make the nullness checks explicit).